src/qubex/experiment/mixin/characterization_mixin.py

```
@@ -878,9 +896,9 @@ def t2_sequence(T: int) -> PulseSchedule:
 
     def ramsey_experiment(
         self,
-        targets: Collection[str] | None = None,
+        targets: Collection[str] | str | None = None,
         *,
-        time_range: ArrayLike = np.arange(0, 20001, 100),
+        time_range: ArrayLike = np.arange(0, 10_001, 100),
```

Ruff (0.8.2)

Do not perform function call `np.arange` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)